lib/aruba/cucumber/core.rb

```diff
@@ -1,4 +1,4 @@
-if Aruba::VERSION >= '1.0.0'
+if false && Aruba::VERSION >= '1.0.0'
```

Comment:
There are several cucumber scenario's that check directories. These fail because they expect `tmp/aruba`, and not `tmp/cucumber`. This is fairly easy to change, but to me it makes more sense to let each user of aruba to set their own working directory if necessary, since there are many ways to parallelize the build, e.g., by running several instances of cucumber at the same time.